script_parameters.yml

```diff
@@ -0,0 +1,249 @@
+# This example workflow demonstrates how to pass parameters into different types
+# of scripts:
+#   1) SQL script
+#   2) Container script
+#   3) Python script
+#   4) R script
+#   5) Custom script
+#
+# It generally involves adding a `params` list and an `arguments` hash to the
+# task inputs.  The `params` list defines the names and types of the parameters
+# that the script can take.  The `arguments` hash contains the values for those
+# parameters.  Think of it like a form, where `params` defines the checkboxes
+# and input fields on the form, and `arguments` is filling the form out.
+#
+# Currently, it is not possible to pass parameters from a workflow into an
+# existing script job (using the `civis.run_job` action).  Passing parameters
```

Comment:
Do we want this to be possible? This should just involve work on mistral-civis-actions, right? Shouldn't necessarily be a part of this ticket, but do we want to ticket this work?